modules/git/commit.go

```diff
@@ -248,6 +248,22 @@ func CommitChanges(repoPath string, opts CommitChangesOptions) error {
 	return err
 }
 
+func allCommitsCount(repoPath string) (int64, error) {
+	cmd := NewCommand("rev-list", "--all", "--count")
+
+	stdout, err := cmd.RunInDir(repoPath)
```

Comment:
no need for `cmd` variable if used only once
```suggestion
	stdout, err := NewCommand("rev-list", "--all", "--count").RunInDir(repoPath)
```